tensorboard/plugins/debugger/tf_debugger_dashboard/tf-session-runs-view.html

```diff
@@ -66,6 +69,11 @@
         background-color: rgb(172, 232, 188);
         font-weight: bold;
       }
+
+      .node-or-tensor-element {
+        text-decoration: underline;
+        cursor: pointer;
```

Comment:
Optional: Add a `.node-or-tensor-element:hover` effect to emphasize how an item is clickable.

Reply:
Good suggestion. I did this for all clickable elements currently present in the Debugger Plugin.